File: Java/Car.java
package Java;

public class Car {
    Integer id;
    String license;
    Driver driver;
    Integer passenger;

    public Car(String license, Driver driver) {
        this.license = license;
        this.driver = driver;
    }

    public String DataCar() {
        String strCar = "id: " + id;
        strCar += "\nLicense: " + license;
        strCar += "\nDriver: " + driver.name;
        strCar += "\nPassenger: " + passenger;
        return strCar;
    }
}

File: PHP/Car.php
<?php

class Car
{
    public $id;
    public $license;
    public $driver;
    public $passenger;

    public function __construct($license,$driver)
    {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function DataCar(){
        $strCar = "License: ".$this->license;
        $strCar = $strCar."\nDriver Name: ".$this->driver->name;
        $strCar = $strCar."\nPassenger: ".$this->passenger;
        return $strCar;
    }
}
